### PR TITLE
[IOTDB-4891] Rename storage group to database

### PR DESCRIPTION
**Pull request by qiaojialin**
1. Change some information about storage group to database
2. Unify all ColumnHeader to Camel case with capitalize the first letter, e.g. DataType

### CODE REVIEW

**Review by JackieTien97 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -245,25 +245,25 @@ IoTDB>
...
 
-We can also use SHOW STORAGE GROUP to check the storage group being created:
+We can also use SHOW DATABASES to check the database being created:
 
 ```
-IoTDB> SHOW STORAGE GROUP
+IoTDB> SHOW DATABASES
 +-------------+
-|storage group|
+|     Database|
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply README.md:259]: not problem

File: README_ZH.md [hunk 2]
```diff
@@ -231,21 +231,21 @@ IoTDB>
...
 ```
-IoTDB> SHOW STORAGE GROUP
-+-------------+
-|storage group|
-+-------------+
-|      root.ln|
-+-------------+
+IoTDB> SHOW DATABASES
++--------+
+|Database|
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply README_ZH.md:245]: not problem

File: docs/UserGuide/QuickStart/QuickStart.md [hunk 3]
```diff
@@ -137,25 +137,25 @@ IoTDB>
...
 
-We can also use SHOW STORAGE GROUP to check created storage group:
+We can also use SHOW DATABASES to check created databases:
 
 ```
-IoTDB> SHOW STORAGE GROUP
+IoTDB> SHOW DATABASES
 +-----------------------------------+
-|                      Storage Group|
+|                           Database|
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply docs/UserGuide/QuickStart/QuickStart.md:151]: not problem

File: docs/UserGuide/QuickStart/QuickStart.md [hunk 4]
```diff
@@ -169,7 +169,7 @@ To query the specific timeseries, use SHOW TIMESERIES \<Path\>. \<Path\> represe
 ```
 IoTDB> SHOW TIMESERIES
 +-------------------------------+---------------+--------+--------+
-|                     Timeseries|  Storage Group|DataType|Encoding|
+|                     Timeseries|       Database|DataType|Encoding|
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply docs/UserGuide/QuickStart/QuickStart.md:172]: not problem

File: docs/zh/UserGuide/Maintenance-Tools/Maintenance-Command.md [hunk 5]
```diff
@@ -147,7 +147,7 @@ IoTDB> create timeseries root.ln.d1.s1 with datatype=BOOLEAN,encoding=PLAIN
 Msg: The statement is executed successfully.
 IoTDB> show regions
 +--------+------------+------+-------------+------------+----------+----------+---------+-------+------+
-|RegionId|        Type|Status|storage group|Series Slots|Time Slots|DataNodeId|     Host|RpcPort|  Role|
+|RegionId|        Type|Status|database|Series Slots|Time Slots|DataNodeId|     Host|RpcPort|  Role|
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply docs/zh/UserGuide/Maintenance-Tools/Maintenance-Command.md:150]: fixed , all Database

File: docs/zh/UserGuide/Maintenance-Tools/Maintenance-Command.md [hunk 6]
```diff
@@ -169,7 +169,7 @@ IoTDB> insert into root.ln.d1(timestamp,s1) values(1,true)
...
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply docs/zh/UserGuide/Maintenance-Tools/Maintenance-Command.md:172]: fixed

**Review by HTHou — APPROVED** [review]

**Review by xingtanzjr — APPROVED** [review]

**Pull request merged**